fake_geo_images/raster.py

```diff
@@ -0,0 +1,51 @@
+"""
+Common raster handling methods
+"""
+from pathlib import Path
+
+from rio_cogeo.cogeo import cog_translate
+from rio_cogeo.profiles import cog_profiles
+
+from .logging import get_logger
+
+logger = get_logger(__name__)
+
+
+def to_cog(path_to_image: Path, profile: str = "deflate", **options) -> bool:
+    """
+    Converts a "normal" GeoTIFF into a Cloud-optimized GeoTIFF
+    Args:
+        path_to_image: path to GeoTIFF
+        profile: compression profile
+        options: additional kwargs
+
```

Comment:
We could add `blockutils` as a requirement to this package and then remove both the raster and logging module.

Reply:
Was thinking about that, but it does not feel right. blockutils is a package especially for people building blocks on up42, while this packages has a broader scope, which was the reason to make it separate. Also, think there is the chance that blockutils will use fake-geo-images and a circular dependency which can cause a lot of trouble.

Reply:
Gotcha, makes sense!